src/env/mpicc.bash.in

```diff
@@ -69,8 +69,8 @@ Show=
 # (e.g., "cc -64" becomes "cc--64", that file is sources, allowing other
 # changes to the compilation environment.  See the variables used by the 
 # script (defined above)
-# Added MPICH_CC_OLD, MPICH_CC can be used to prefix CC with external utility,
-# e.g. setenv MPICH_CC 'eval linkcache $MPICH_CC_OLD'
+# Added MPICH_CC_OLD, MPICH_CC can be used to prefix CC
+# with external utility, e.g. setenv MPICH_CC 'eval linkcache $MPICH_CC_OLD'
```

Comment:
Could you split commit separate formatting, white space, and spelling changes into a separate commit?